docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.7"
+services:
+   sync:
+     build: .
+     depends_on:
+       - "mysql"
+     ports:
+# CHANGE TO MATCH YOUR CONFIG/PROXY
```

Comment:
It's a minor thing but could you capitalize these comments normally instead of all caps?